.github/workflows/lint.yml

```diff
@@ -0,0 +1,37 @@
+name: Lint
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php-versions: ['7.3']
```

Comment:
Shouldn't the linter run for the lowest php version we support?

Reply:
Why should it? The linter is a packagist package, which is installed via composer. The PHP version shouldn't matter.

Reply:
When the version doesn't matter it's fine.